### ISSUE TITLE
Website: make 1.0 post mobile-friendly

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#617]
https://quick-lint-js.com/blog/version-1.0/

**Comment by lifeinData** [comment]
I claim this for-hire task

**Comment by strager** [comment]
https://quick-lint-js.com/blog/version-2.0/ is also affected.

**Comment by strager** [comment]
@lifeinData Are you going to work on this task?

**Comment by lifeinData** [comment]
@strager Yes sir! I will indeed do that. Amazon offer fell through so now I'm getting back to doing more dev tings

**Comment by leszek888** [comment]
I could take this one. From what I can see, the main problem are the Eslint/quick-lint-js comparisons.
